src/domains/mapDomain.ml

```diff
@@ -65,6 +65,59 @@ sig
   val trace_enabled: bool (* Just a global hack for tracing individual variables. *)
 end
 
+(** Subsignature of {!S}, which is sufficient for {!Print}. *)
+module type Bindings =
+sig
+  type t
+  type key
+  type value
+  val bindings: t -> (key * value) list
+  val iter: (key -> value -> unit) -> t -> unit
+end
+
+(** Reusable output definitions for maps. *)
+module Print (D: Groupable) (R: Printable.S) (M: Bindings with type key = D.t and type value = R.t) =
+struct
+  let show x = "mapping" (* TODO: WTF? *)
```

Comment:
Address TODO?

Reply:
For some odd reason `MapDomain.pretty` uses this `show` just as a prefix to distinguish itself from sets: `mapping {...}` vs `{...}`. So far this weird `show` hasn't been an issue, so I wouldn't bother refactoring it now since it'll all end up refactored for #213.